pom.xml

```diff
@@ -433,6 +433,10 @@
             <groupId>com.sun.jersey</groupId>
             <artifactId>jersey-server</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>io.netty</groupId>
```

Comment:
This exclusion from test jar should be fine as its only minor version changes. Usually netty 4.1.x series is backward compatible.